app/src/test/java/org/mozilla/focus/browser/integration/FullScreenIntegrationTest.kt

```diff
@@ -95,61 +99,58 @@ internal class FullScreenIntegrationTest {
         val activity: Activity = mock()
         doReturn(activityWindow).`when`(activity).window
         doReturn(decorView).`when`(activityWindow).decorView
-        val integration = FullScreenIntegration(
-            activity, mock(), null, mock(), mock(), mock(), mock(), mock()
+
+        val integration = spy(
```

Comment:
Don't think we need a `spy` here since we are not mocking or verifying any of `FullScreenIntegration`'s properties and methods?